README.md

```diff
@@ -1,5 +1,129 @@
 
-# Specification 'awesome-semantic-shapes'
+> A curated list of Semantic Shapes resources. Contributions welcome!
+> Please, read the [Contribution Guidelines](CONTRIBUTING.md) first.
 
-This is the repository for awesome-semantic-shapes. You're welcome to contribute! Let's make the Web rock our socks
-off!
+Semantic shapes enable you to validate RDF graphs against a set of conditions.
+Semantic shapes can also be viewed as a description of data graphs that satisfy these conditions.
+Such descriptions may be used for a variety of purposes besides validation,
+including user interface building, code generation, and data integration.
+Semantic shapes are frequently described using the SHACL or ShEx language.
+
+## Contents
+
+- [SHACL Validators](#shacl-validators)
+- [ShEx Validators](#shex-validators)
+- [Shapes Discovery Tools](#shapes-discovery-tools)
+- [Shapes Collections](#shapes-collections)
+- [Shape Conversion Tools](#shape-conversion-tools)
+- [Shape Generators](#shape-generators)
+- [Shape-based Query Generators](#shape-based-query-generators)
+- [Shape Editors, Visualizations](#shape-editors-visualizations)
+- [Declarative UIs](#declarative-uis)
+- [IDE support](#ide-support)
+- [Books](#books)
+- [Tutorials](#tutorials)
+- [Talks](#talks)
+- [Presentations](#presentations)
+- [Specifications](#specifications)
+- [Legend](#legend)
+
+## SHACL Validators
+
+- [Apache Jena SHACL](https://github.com/apache/jena/tree/main/jena-shacl) <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/apache/jena" align="top"> - Supports SHACL Core, SHACL-SPARQL; `Apache-2.0` license; `Java`.
+- [RDF4J SHACL Engine](https://github.com/eclipse-rdf4j/rdf4j/tree/main/core/sail/shacl) <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/eclipse-rdf4j/rdf4j" align="top"> - Supports SHACL Core (without some property paths), SHACL-SPARQL, incremental validation; `BSD-3-Clause` license; `Java`.
+- [rdf-validate-shacl](https://github.com/zazuko/rdf-validate-shacl) <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/zazuko/rdf-validate-shacl" align="top"> - Supports SHACL Core; pure JavaScript validator on top of the [RDFJS](https://rdf.js.org/) stack; [playground](https://zazuko.github.io/shacl-playground); `MIT` license; `JavaScript`.
+- [SHACL for Ruby](https://github.com/ruby-rdf/shacl) ![GitHub last commit](https://img.shields.io/github/last-commit/ruby-rdf/shacl) - A pure-Ruby library for working with the Shape Constraint Language to validate the shape of RDF graphs; `BSD-3-Clause` license; `Ruby`.
+- [shacl-engine](https://github.com/rdf-ext/shacl-engine) <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/rdf-ext/shacl-engine" align="top"> - Supports SHACL Core, SHACL-SPARQL; A fast engine for data provided as [RDF/JS](http://rdf.js.org/data-model-spec/) objects; [playground](https://playground.rdf-ext.org/shacl/); `MIT` license; `JavaScript`.
```

Comment:
The command line tool [rdf-ext-cli](https://github.com/rdf-ext/rdf-ext-cli) provides a validation feature based on shacl-engine. Data and shapes can be given as file, URL, or SPARQL endpoint + query.